awscli/botocore/data/ssm-sap/2018-05-10/paginators-1.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We should make sure to delete the old `ssmsap` directory as part of this PR, including the old `ssmsap` endpoint tests.